serializers/postgres.rb

```diff
@@ -12,6 +12,7 @@ def self.serialize_internal(pg, options = {})
       storage_size_gib: pg.storage_size_gib,
       target_storage_size_gib: pg.target_storage_size_gib,
       version: pg.version,
+      current_version: pg.current_version,
```

Comment:
When adding a serializer column, update the SDK (`sdk/ruby/lib/ubicloud/model/postgres.rb`) and CLI (`cli-commands/pg/post/show.rb`) as well.